### PR TITLE
Angular material added and bootstrap removed

### LINKED ISSUE #14: Use Angular Material for all components.

### PR DESCRIPTION
**Pull request by nityanandagohain**
fixes #14 

Deployment link :-http://cagey-oatmeal.surge.sh/

### CODE REVIEW

**Comment by nityanandagohain** [comment]
@gabru-md  I have removed ngbootstrap form terminal and tree component.

File: git-visualize/package.json [hunk 1]
```diff
@@ -12,16 +12,17 @@
...
     "@angular/common": "^5.2.0",
     "@angular/compiler": "^5.2.0",
     "@angular/core": "^5.2.0",
     "@angular/forms": "^5.2.0",
     "@angular/http": "^5.2.0",
+    "@angular/material": "^5.2.5",
     "@angular/platform-browser": "^5.2.0",
     "@angular/platform-browser-dynamic": "^5.2.0",
     "@angular/router": "^5.2.0",
-    "@ng-bootstrap/ng-bootstrap": "^1.1.1",
```
[the remark this answers is not in the document]
Comment (nityanandagohain) [reply git-visualize/package.json]: :+1:

**Comment by gabru-md** [comment]
@harshit98 we may need a review here please

**Comment by gabru-md** [comment]
I reviewed the deployment link, and there is some problem,
The page seems to go out of view. It must not be scrollable. The page must fit completely with the dimensions of the web browser.
Can you please check onto this?

**Comment by nityanandagohain** [comment]
sure

**Comment by gabru-md** [comment]
@nityanandagohain please update?

**Comment by nityanandagohain** [comment]
@gabru-md  I have updated it along with deployment link but still build is failing.

**Comment by gabru-md** [comment]
Thank You @nityanandagohain for your PR,

But the ReactJS frontend has arrived. Maybe you can look into it for working.
Thanks for your contribution though, Please keep contributing :) :D

**Pull request closed without merging**